src/transforms/scale.jl

```diff
@@ -38,7 +38,7 @@ assertions(::Type{Scale}) = [assert_continuous]
 isrevertible(::Type{Scale}) = true
 
 function colcache(transform::Scale, x)
-  levels = (transform.low, transform.high)
+  levels = map(eltype(x), (transform.low, transform.high))
```

Comment:
I started thinking that maybe we shouldn't change the code here, but instead ask users to create a Scale transform with the appropriate type. `Scale(0.0f0, 1.0f0)` if they want to preserve the result type.

@eliascarv what do you think? I feel that converting the type explicitly like in the proposed change can lead to more issues when the eltype of the column is not trivial. For example, if the eltype is Integer, we will loose precision, etc.

Reply:
So I think a better fix would consist of adding a type parameter to Scale{T}.

Reply:
I think this problem deserves a more general solution for all Colwise.

Reply:
That is a good point as well. I think we can still improve the situation with the Scale transform. It is too rigid at the moment requireing Float64 fields. @antholzer can you parametrize the transform to accept low::T and high::T?

Reply:
I added a type parameter `T`.

For `MinMax` I used `low=0, high=1` instead of `low=0.0, high=1.0` since we do not lose precision and this way all Vectors with `Float16`, `Float32`, `Float64` or `Int`entries do not get converted.

Reply:
Awesome @antholzer , thank you for the great improvements. Will leave another review now.